.github/workflows/main.yaml

```diff
@@ -0,0 +1,45 @@
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
This will run on pushes to `master` *and* PRs to `master`, correct?  

The `build` job below deploys to GH Pages, but the previous CI configuration only deployed to GH Pages on pushes to master.  And it only built the Nikola page on PRs to `master`.

Reply:
> The build job below deploys to GH Pages, but the previous CI configuration only deployed to GH Pages on pushes to master. And it only built the Nikola page on PRs to master.

The site builds are activated for both PRs to master and pushes to master, but the deployment only happens if it is a push to master... It's my understanding that this is similar to what we used to do on CircleCI.